src/reducers/profile-view.js

```diff
@@ -559,43 +557,23 @@ export const getRightClickedThreadIndex = (state: State) =>
 export const getSelection = (state: State) =>
   getProfileViewOptions(state).selection;
 
-export type SelectorsForThread = {
-  getThread: State => Thread,
-  getViewOptions: State => ThreadViewOptions,
-  getTransformStack: State => TransformStack,
-  getTransformLabels: State => string[],
-  getRangeFilteredThread: State => Thread,
-  getRangeAndTransformFilteredThread: State => Thread,
-  getJankInstances: State => TracingMarker[],
-  getProcessedMarkersThread: State => Thread,
-  getTracingMarkers: State => TracingMarker[],
-  getTracingMarkersForView: State => TracingMarker[],
-  getMarkerTiming: State => MarkerTimingRows,
-  getRangeSelectionFilteredTracingMarkers: State => TracingMarker[],
-  getRangeSelectionFilteredTracingMarkersForHeader: State => TracingMarker[],
-  getFilteredThread: State => Thread,
-  getRangeSelectionFilteredThread: State => Thread,
-  getCallNodeInfo: State => CallNodeInfo,
-  getCallNodeMaxDepth: State => number,
-  getSelectedCallNodePath: State => CallNodePath,
-  getSelectedCallNodeIndex: State => IndexIntoCallNodeTable | null,
-  getExpandedCallNodePaths: State => PathSet,
-  getExpandedCallNodeIndexes: State => Array<IndexIntoCallNodeTable | null>,
-  getCallTree: State => CallTree.CallTree,
-  getStackTimingByDepth: State => StackTiming.StackTimingByDepth,
-  getCallNodeMaxDepthForFlameGraph: State => number,
-  getFlameGraphTiming: State => FlameGraph.FlameGraphTiming,
-  getFriendlyThreadName: State => string,
-  getThreadProcessDetails: State => string,
-  getSearchFilteredMarkers: State => MarkersTable,
-  unfilteredSamplesRange: State => StartEndRange | null,
-};
-
-const selectorsForThreads: { [key: ThreadIndex]: SelectorsForThread } = {};
+const selectorsForThreads: { [key: ThreadIndex]: * } = {};
```

Comment:
This is the only location where I wasn't so sure it would work, but I tried it and it works, so... yay :)